src/cf-forms.less

```diff
@@ -197,26 +217,30 @@ textarea {
 
 .@{cf-forms_input-icon-class} {
     @font-size: 20px;
-
-    position: relative;
-    top: unit(6px / @font-size, em);
-    margin-left: unit(4px / @font-size, em);
     font-size: unit(@font-size / @base-font-size-px, em);
 }
 
+.form-label-error .@{cf-forms_input-icon-class} {
+    color: @input-error;
+}
+
+// Deprecated
```

Comment:
Anyone know if you can send warnings to the console with Less like you can in Sass with [@warn](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_6)?